README.md

```diff
@@ -18,18 +18,34 @@ Our goal is to establish a new social norm in the tech industry of companies pay
 of $2000 per year per developer on staff at our companies, to Open Source maintainers of each company’s choosing. This
 is in addition to the code, developer time, and gifts-in-kind that many of our companies also contribute.
 
-* **Want to learn more?** Visit [opensourcepledge.com][osp] and follow our progress on our [launch
-  ticket][launch-ticket].
+* **Want to learn more?** Visit [opensourcepledge.com][osp].
 * **Want to join the Pledge?** [Find out how][join].
 * **Want to participate as a maintainer, foundation or funding platform?** [Let us know!][newissue]
-* **Want to help?** Read [CONTRIBUTING.md](CONTRIBUTING.md), [dive into our issues][issues], and feel free to reach out
-  to the working group leads either in issues or [on Discord][discord].
+* **Want to learn about the Pledge's background?** Read:
+  * [“The Open Source Sustainability Crisis”][crisis] by [Chad][openpath]
+  * [“Open Source Is a Restaurant”][restaurant] by [Chad][openpath]
```

Comment:
Let's use David's post on the Sentry blog rather than this one.

Reply:
Done!